### ISSUE TITLE
Need our own database to store scraped pastes

### ISSUE THREAD

**Issue by i02132002** [xofbd/pastebin#10]
Since we're scraping the website, we probably need to store the scraped data locally. Since we'll need to scrape periodically, there needs to be some kind of pipeline to feed the scraped data into the database. How should we go about this?

**Comment by xofbd** [comment]
Yes, at some point we need think about how to best store the data. We can cross that bridge when we get there, though
